### ISSUE TITLE
about results

### ISSUE THREAD

**Issue by lsj910128** [nqanh/affordance-net#13]
Hi, do you know why the segmented result (the area of the target object) is always smaller than the ground truth? Someone told me the reason is threshold for the segmentation mask in the method is too large. But I don't what is this threshold?

**Comment by lsj910128** [comment]
Hi, I have a question about Robust Resizing Affordance Mask: how to set the hyperparameter α and where to set?  How can I make α learned during the network training phase.

**Comment by nqanh** [comment]
During training, alpha is 0.005 (line `478` and `485` in `proposal_target_layer.py`). During testing, you can change line 20 in `demo_img.py`.

This param is not learnable.

**Comment by lsj910128** [comment]
Thank you for your reply.
But how to determine the value of alpha?

**Comment by nqanh** [comment]
It's a hyperparameter. Setting a higher value of alpha will lead to noisier affordance map. You may want to play with this param in the demo script.
